### PR TITLE
Packages: Add "gutenberg" to the list of keywords in package.json

### PR DESCRIPTION
**Pull request by gziolo**
## Description
Suggested by @mtias in the comments under Matt's post after WordCamp EU:
https://ma.tt/2020/06/wceu-questions/#comment-590029

> > I was naively searching npm for Gutenberg. Should have thought to search WordPress.
> We should probably add the “gutenberg” keywords somewhere in those packages.

### CODE REVIEW

File: packages/a11y/package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
-		"a11y",
+		"gutenberg",
+		"accessibility",
```
Comment (nosolosw) [inline packages/a11y/package.json:10]:
```suggestion
		"accessibility",
		"a11y",
```

Comment (nosolosw) [reply packages/a11y/package.json:10]: Can we leave `a11y`? It seems a [common label](https://www.npmjs.com/search?q=a11y).

File: packages/babel-plugin-import-jsx-pragma/package.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
-		"babel-plugin",
+		"gutenberg",
+		"babel",
+		"plugin",
```
Comment (nosolosw) [inline packages/babel-plugin-import-jsx-pragma/package.json:11]:
```suggestion
		"plugin",
		"babel-plugin",
```

babel-plugin also seems a [common label](https://www.npmjs.com/search?q=keywords:babel-plugin)

File: packages/babel-preset-default/package.json [hunk 3]
```diff
@@ -6,9 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"babel",
-		"preset",
-		"default"
+		"preset"
```
Comment (nosolosw) [inline packages/babel-preset-default/package.json:11]:
```suggestion
		"preset",
		"babel-preset"
```

File: packages/custom-templated-path-webpack-plugin/package.json [hunk 4]
```diff
@@ -6,8 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"webpack",
-		"webpack-plugin"
+		"plugin"
```
Comment (nosolosw) [inline packages/custom-templated-path-webpack-plugin/package.json:11]: webpack-plugin seems a [common label](https://www.npmjs.com/search?q=keywords:webpack-plugin) as well

```suggestion
		"plugin",
		"webpack-plugin",
```

File: packages/library-export-default-webpack-plugin/package.json [hunk 5]
```diff
@@ -6,8 +6,10 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"webpack",
-		"webpack-plugin"
+		"plugin",
```
Comment (nosolosw) [inline packages/library-export-default-webpack-plugin/package.json:11]:
```suggestion
		"plugin",
		"webpack-plugin",
```

it looks like `webpack-plugin` is something people use, doesn't hurt having it

**Comment by gziolo** [comment]
Thanks @nosolosw, all suggestions applied :)

File: packages/postcss-themes/package.json [hunk 6]
```diff
@@ -6,9 +6,10 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"postcss",
 		"css",
-		"postcss-plugin",
+		"plugin",
```
Comment (nosolosw) [inline packages/postcss-themes/package.json:12]:
```suggestion
		"plugin",
		"postcss-plugin",
```

apparently, [postcss-plugin](https://www.npmjs.com/search?q=keywords:postcss-plugin) is a thing

File: packages/scripts/package.json [hunk 7]
```diff
@@ -6,9 +6,9 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"scripts",
```
Comment (nosolosw) [inline packages/scripts/package.json:10]:
```suggestion
		"scripts"
```

File: packages/scripts/package.json [hunk 8]
```diff
@@ -6,9 +6,9 @@
...
-		"reusable-scripts",
-		"wordpress-scripts"
+		"reusable-scripts"
```
Comment (nosolosw) [inline packages/scripts/package.json:11]:
```suggestion
```

only 2 results found for this label

**Review by oandregal — APPROVED** [review]

**Comment by mtias** [comment]
Thanks :)

**Pull request merged**